src/main/resources/io/jenkins/plugins/cloudevents/CloudEventsConfig/config.jelly

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+   <f:section title="${%CloudEvents Plugin}">
+    <f:entry title="Configure Event Endpoints" field="endpoints">
+        <f:repeatable var="endpointsList" name="endpoints" add="${%Add Endpoint}" items="${instance.endpoints}">
+            <fieldset>
+
+                    <f:entry field="sinkURL" title="Sink URL" description="Enter the URL of the sink to send events to.">
+                    <f:textbox value="${endpointsList.sinkURL}"/>
+                    </f:entry>
+
+                    <f:entry title="Event" description="Select events for which you want to receive CloudEvents" field="event">
+                       <select class="setting-input" name="event">
```

Comment:
We should be able to populate this programatically. Check this example [here].(https://www.jenkins.io/doc/developer/forms/jelly-form-controls/#select-drop-down-menu-with-model-filled-values)